tests/unit_tests/derived_model/test_derived_model.py

```diff
@@ -93,7 +92,7 @@ def check_zone_prop(z_prop):
                                                null_value = -1)
     assert za_uuid is not None
 
-    # add a zone by layer property based on the neat cells property
+    #  add a zone by layer property based on the neat cells property
```

Comment:
is this ugly double space at the start of a comment another python black thing sneaking in?

Reply:
Seems to be the default PyCharm formatter because my yapf plugin isn't working